components/LandingPage/CreateButtons.tsx

```diff
@@ -1,15 +1,24 @@
-import { baseSepolia } from 'viem/chains'
+import { base, baseSepolia, zora, zoraSepolia } from 'viem/chains'
 import CreateButton from '../CreateButton'
 import { useParams } from 'next/navigation'
 
 const CreateButtons = () => {
   const params = useParams()
   const collection = params.collection as string
-  const chainId = params.chainId
+  const chainId = Number(params.chainId)
+
+  const isBase = chainId === baseSepolia.id || chainId === base.id
+  const isZora = chainId === zoraSepolia.id || chainId === zora.id
+
+  const previewChainId = () => {
+    if (isBase) return baseSepolia.id
+    if (isZora) return zoraSepolia.id
+    return baseSepolia.id
+  }
```

Comment:
preview button only exists for `new contract` right?
- actual: ziad adding `new contract support` on zora
- expected: only new contract support on `base`